semantic-python/src/Language/Python/Core.hs

```diff
@@ -97,9 +96,8 @@ compile :: ( Compile py
         -> m (t Name)
 compile t = compileCC t (pure none)
 
-locFromTSSpan :: SourcePath -> Source.Span -> Loc
-locFromTSSpan fp (Source.Span (Source.Pos a b) (Source.Pos c d))
-  = Data.Loc.Loc (rawPath fp) (Data.Loc.Span (Data.Loc.Pos a b) (Data.Loc.Pos c d))
+locFromTSSpan :: SourcePath -> Span -> Loc
+locFromTSSpan fp = Data.Loc.Loc (rawPath fp)
```

Comment:
I want to change how annotations work so as to not require this sort of thing but for now this will do.